appveyor.yml

```diff
@@ -38,9 +38,9 @@ test_script:
     - set RUSTFLAGS=-g
     - set RUST_BACKTRACE=1
     - cargo build --release --all-targets --all-features
-    - cargo test --release
+    - cargo test --release --all-targets
```

Comment:
The PR title says `--all-features`, while this says `--all-targets`

Reply:
*oops*